gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -1622,29 +1685,52 @@ gst_tensor_converter_parse_caps (GstTensorConverter * self,
 
   /** set the number of frames in dimension */
   if (frames_dim >= 0) {
-    config.info.dimension[frames_dim] = self->frames_per_tensor;
+    config.info.info[0].dimension[frames_dim] = self->frames_per_tensor;
   }
 
-  if (!gst_tensor_config_validate (&config)) {
+  if (!gst_tensors_config_validate (&config)) {
     /** not fully configured */
-    GST_ERROR_OBJECT (self, "Failed to configure tensor info.\n");
+    GST_ERROR_OBJECT (self, "Failed to configure tensors info.\n");
     return FALSE;
   }
-
   if (gst_tensor_info_validate (&self->tensor_info)) {
     /** compare tensor info */
-    if (!gst_tensor_info_is_equal (&self->tensor_info, &config.info)) {
+    if (!gst_tensor_info_is_equal (&self->tensor_info, &config.info.info[0])) {
       GST_ERROR_OBJECT (self, "Failed, mismatched tensor info.\n");
       return FALSE;
     }
   }
 
   self->in_media_type = in_type;
-  self->tensor_configured = TRUE;
-  self->tensor_config = config;
+  self->tensors_configured = TRUE;
+  self->tensors_config = config;
   return TRUE;
 }
 
+/**
+ * @brief Get tensor config from tensors config
+ */
+static void
+gst_tensor_config_from_tensors (GstTensorConfig * tensor_config,
+    GstTensorsConfig * tensors_config)
+{
+  tensor_config->info = tensors_config->info.info[0];
+  tensor_config->rate_n = tensors_config->rate_n;
+  tensor_config->rate_d = tensors_config->rate_d;
+}
+
+/**
+ * @brief Get converter's subplugins name
+ */
+static const gchar *
+getExternalConverterName (const char *name)
+{
+  if (g_strcmp0 (name, "other/flatbuf-tensor") == 0) {
```

Comment:
plz mark @todo compare mimetype and return lib name from sub-plugin (not hardcoded value)

Reply:
But the sub-plugin is not registered yet at this line. You mean `nnstreamer_subplugin.c`?